api/connect_ca_test.go

```diff
@@ -77,7 +76,9 @@ func TestAPI_ConnectCAConfig_get_set(t *testing.T) {
 		}
 		parsed, err := ParseConsulCAConfig(conf.Config)
 		r.Check(err)
-		verify.Values(r, "", parsed, expected)
+		if !assert.Equal(t, expected, parsed) {
```

Comment:
Since we are changing it why not just use:

`require.Equal(t, expected, parsed)`

No need for the if and FailNow then.

Reply:
Because we are in a retry block here and we want to retry instead of aborting the test, which is what require would do.

Reply:
Sorry, what you actually want then is

`require.Equal(r, expected, parsed)`

It will call the right methods on the retry.R. I have done that a number of places in more recent tests.